pallets/tinlake-investor-pool/src/tests.rs

```diff
@@ -316,7 +316,7 @@ fn epoch() {
 		let pool = TinlakeInvestorPool::pool(0).unwrap();
 		assert_eq!(
 			pool.tranches[0].interest_per_sec,
-			Perquintill::from_parts(000000003170979198)
+			Rate::from_inner(1_000000003170979198376458650)
```

Comment:
Where is the number coming from?

Reply:
This is 10% annual interest rate divided by (365 days * 24 hours * 60 minutes * 60 seconds).

The changes from the old test here are:
* Nine more digits of precision
* Adding a one to the front in storage, since previously the first thing we did when reading from storage was add 1